jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	coverageProvider: 'v8',
```

Comment:
```suggestion
	coverageProvider: 'v8',
	coverageReporters: ['lcov', 'text-summary'],
```

Not saying this will fix the failing CI but just maybe it will. Lcov is needed for coveralls, text-summary is for STDOUT.